hack/test

```diff
@@ -22,6 +22,7 @@ set -eu -o pipefail
 : "${TEST_DOCKERD_BINARY=$(which dockerd)}"
 : "${TEST_REPORT_SUFFIX=}"
 : "${TEST_KEEP_CACHE=}"
+: "${TESTFLAGS:=}"
```

Comment:
`:=` intentional in here?

Reply:
Yes

```shell
set -u;
: "${FOO:-}"
echo $FOO
bash: FOO: unbound variable
```

Reply:
Oh sorry, I guess we don't need the `:`, right.

Reply:
Updated.